### ISSUE TITLE
sql type expansion.

### ISSUE THREAD

**Issue by RossRogers** [mhallin/juniper-example-todo-backend#2]
Magnus,

Can you expand this example to do Datetimes, nullable datetimes, one-to-one foreign keys, one to many foreign keys and maybe even against the mysql backend :-P

Thanks if you can.

**Comment by RossRogers** [comment]
Mr. Herzog ( @theduke ) ,

Any chance you might be up for expanding this example?  I'm still floundering trying to get a basic test case working. 

@mhallin ,

I really appreciate all your work making juniper, but if there isn't a good starter repo for people, it may take longer for others to start using your code base.  I, personally, really want to, but the friction with wrestling with the Rust compiler as a newbie is stymieing me.

- Ross

**Comment by theduke** [comment]
@RossRogers I agree that it's super important, I hope I'll have time this weekend.

**Comment by RossRogers** [comment]
@theduke,

Far too kind, sir.  Thank you.
